includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -17,7 +17,7 @@
  *
  * Allowed tags array is generated from this protocol buffer:
  *
- *     https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii
+ *     https://github.com/ampproject/amphtml/blob/bd29b0eb1b28d900d4abed2c1883c6980f18db8e/validator/validator-main.protoascii
```

Comment:
Yeah, that's true, though there's also some value of having the exact snapshot in time referenced in the comment. As then we can check to see if things have changed since then.